### ISSUE TITLE
Node.js Foundation Security WorkGroup Meeting 2019-12-30

### ISSUE THREAD

**Issue by mhdawson** [nodejs/security-wg#615]
## Time

**UTC Mon 30-Dec-2019 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Mon 30-Dec-2019 08:00 (08:00 AM) |
| US / Mountain | Mon 30-Dec-2019 09:00 (09:00 AM) |
| US / Central  | Mon 30-Dec-2019 10:00 (10:00 AM) |
| US / Eastern  | Mon 30-Dec-2019 11:00 (11:00 AM) |
| London        | Mon 30-Dec-2019 16:00 (04:00 PM) |
| Amsterdam     | Mon 30-Dec-2019 17:00 (05:00 PM) |
| Moscow        | Mon 30-Dec-2019 19:00 (07:00 PM) |
| Chennai       | Mon 30-Dec-2019 21:30 (09:30 PM) |
| Hangzhou      | Tue 31-Dec-2019 00:00 (12:00 AM) |
| Tokyo         | Tue 31-Dec-2019 01:00 (01:00 AM) |
| Sydney        | Tue 31-Dec-2019 03:00 (03:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Security%20WorkGroup+Meeting+2019-12-30&iso=20191230T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+Dec+30%2C+2019+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1-mlZ5U2bxu_1-SvXRJr_nFRI1gXxtkhva1OWhxZjB-8/edit>

## Agenda

Extracted from **security-wg-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/security-wg

* Nominating Daniel Ruf to the Ecosystem Triage team [#614](https://github.com/nodejs/security-wg/issues/614)
* Triage Buckets and Prioritization [#604](https://github.com/nodejs/security-wg/issues/604)
* Nominate Thomas Watson as WG member [#581](https://github.com/nodejs/security-wg/pull/581)
* Reaching out to other projects of the OpenJS Foundation [#511](https://github.com/nodejs/security-wg/issues/511)

## Invited

* Security wg team: @nodejs/security-wg

## Notes

The agenda comes from issues labelled with `security-wg-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/405474359
* For those who just want to watch We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up. 
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

**Comment by vdeturckheim** [comment]
I will be off on that day and will have to miss this one.

**Comment by lirantal** [comment]
@mhdawson in another issue thread me and @sam-github briefly mentioned canceling this one and catching up on it on the next month due to many people probably being off due to holidays.

**Comment by sam-github** [comment]
I can make it, but if @lirantal and @vdeturckheim cannot, we should skip. Or re-schedule for just one week later?

**Comment by lirantal** [comment]
I personally prefer to move by a week or until the next one

**Comment by MarcinHoppe** [comment]
I think we have a consensus to cancel this WG meeting?

**Comment by sam-github** [comment]
We do
